test/e2e/specs/testdriver.go

```diff
@@ -48,7 +48,8 @@ type GCSFuseCSITestDriver struct {
 	ClientProtocol              string
 	skipGcpSaTest               bool
 	EnableHierarchicalNamespace bool
-	EnableZB                    bool // Enable Zonal Buckets
+	EnableZB                    bool             // Enable Zonal Buckets
+	storageService              *storage.Service // Client to interact with GCS
```

Comment:
Why do we need another storageService? I believe storageServiceManager wraps the service